src/wp-includes/comment.php

```diff
@@ -417,6 +417,7 @@ function get_comment_count( $post_id = 0 ) {
 		'count'                     => true,
 		'update_comment_meta_cache' => false,
 		'orderby'                   => 'none',
+		'type__not_in'              => array( 'note' ),
```

Comment:
This is necessary to filter out Note-type comments from the dashboard. Otherwise, the dashboard shows unintended counts.

<img width="1076" height="449" alt="Image" src="https://github.com/user-attachments/assets/a67c817b-b310-4b89-b289-6318c15e1b64" />